app/controllers/planning_applications/assessment/local_policy_areas_controller.rb

```diff
@@ -0,0 +1,69 @@
+# frozen_string_literal: true
+
+module PlanningApplications
+  module Assessment
+    class LocalPolicyAreasController < AuthenticationController
+      before_action :set_planning_application
+      before_action :set_local_policy, only: %i[show edit update new]
+      before_action :set_local_policy_area, only: %i[edit show]
+
+      def show
+        set_local_policy_area
+      end
+
+      def new
+        @local_policy_area = LocalPolicyArea.new
+      end
+
+      def edit
+        set_local_policy_area
+      end
+
+      def destroy
+        @local_policy_area = LocalPolicyArea.find(params[:id])
```

Comment:
We should always scope our finds through the association otherwise you can walk through id values to find other records. Not particularly sensitive information here but it's good practice to follow.